### PR TITLE
#drone v1.6.0 new features user guide

### PR DESCRIPTION
**Pull request by ElaineSimoes**
View Range, Template Transfer, View Template and Filter by Levels documentation.

### CODE REVIEW

File: docs/pages/ViewManager/VM-Template-Transfer.md [hunk 1]
```diff
@@ -18,29 +18,29 @@ Manage all the Views in your project, as well as View Templates and View Range s
...
+The Template Transfer allows you to copy the View Template properties from several View Templates.
 
 ![Template Transfer](../../../assets\images\ViewManager\VM-Tt-Tab.gif)
 
 <sub>Note: the version on the image may not reflect the [latest version of ViewManager/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
 
 ## Select Origin Template
 
-Choose one Origin View Template to copy their properties.
+Select a Origin View Template to copy its properties.
```
Comment (ViniciusKawazoe) [inline docs/pages/ViewManager/VM-Template-Transfer.md:29]: "Select **an** Origin View"

Comment (ViniciusKawazoe) [inline docs/pages/ViewManager/VM-Template-Transfer.md:29]: "Select **an** Origin View"

File: assets/images/ViewManager/VM-Tt-Destination.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Tt-Destination.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI and install DiStem so it show in Revit's UI.

File: assets/images/ViewManager/VM-Tt-EditViewTemplate.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Tt-EditViewTemplate.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI and install DiStem so it shows in Revit's UI.

File: assets/images/ViewManager/VM-Tt-OriginTemplate.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Tt-OriginTemplate.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI and install DiStem so it shows in Revit's UI.

File: assets/images/ViewManager/VM-Tt-Properties.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Tt-Properties.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI and install DiStem so it shows in Revit's UI.

File: assets/images/ViewManager/VM-Tt-Tab.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Tt-Tab.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI and install DiStem so it shows in Revit's UI.

File: docs/pages/ViewManager/VM-TemplateTransfer.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+On this table you are able to choose one or multiple properties from the Origin View Template, to copy to another one.
+
+1. Use the search box to search by view template properties.
+
+2. Use the checkboxes to select properties.
+
+```yaml
+# Note:  
+Check the Hide un-checked properties to see only the choosen properties.
```
Comment (ViniciusKawazoe) [inline docs/pages/ViewManager/VM-TemplateTransfer.md]: "Check the "Hide ~~un-checked~~ unchecked properties" to see only the ~~choosen~~ selected properties."

File: assets/images/OneFilter/OF-Lv-Isolate.gif
Comment (ViniciusKawazoe) [inline assets/images/OneFilter/OF-Lv-Isolate.gif:1]: The "Levels" tab in the GIFs does not have the "Premium" icon. Please update it.

File: assets/images/OneFilter/OF-Lv-Reset.gif
Comment (ViniciusKawazoe) [inline assets/images/OneFilter/OF-Lv-Reset.gif:1]: The "Levels" tab in the GIFs does not have the "Premium" icon. Please update it.

File: assets/images/OneFilter/OF-Lv-Rule.gif
Comment (ViniciusKawazoe) [inline assets/images/OneFilter/OF-Lv-Rule.gif:1]: The "Levels" tab in the GIFs does not have the "Premium" icon. Please update it.

File: assets/images/OneFilter/OF-Lv-Search.gif
Comment (ViniciusKawazoe) [inline assets/images/OneFilter/OF-Lv-Search.gif:1]: The "Levels" tab in the GIFs does not have the "Premium" icon. Please update it.

File: assets/images/OneFilter/OF-Lv-Selection.gif
Comment (ViniciusKawazoe) [inline assets/images/OneFilter/OF-Lv-Selection.gif:1]: The "Levels" tab in the GIFs does not have the "Premium" icon. Please update it.

File: assets/images/OneFilter/OF-Lv-Views.gif
Comment (ViniciusKawazoe) [inline assets/images/OneFilter/OF-Lv-Views.gif:1]: The "Levels" tab in the GIFs does not have the "Premium" icon. Please update it.

File: assets/images/OneFilter/OF-Lv-Visualize.gif
Comment (ViniciusKawazoe) [inline assets/images/OneFilter/OF-Lv-Visualize.gif:1]: The "Levels" tab in the GIFs does not have the "Premium" icon. Please update it.

File: assets/images/OneFilter/OF-Lv-WholeModel.gif
Comment (ViniciusKawazoe) [inline assets/images/OneFilter/OF-Lv-WholeModel.gif:1]: The "Levels" tab in the GIFs does not have the "Premium" icon. Please update it.

File: docs/pages/OneFilter/OF-Levels.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+The Levels tab allows the user to select categories of elements to find between Levels and Reference Plane.
+
+## Select Category
+
+1. The first step is to select whether you want to apply filters based on the Whole Model or the Active View. Select the radio button to choose the view.
+
+![OneFilter select model view](../../../assets\images\OneFilter\OF-Lv-WholeModel.gif)  
+<sub>Note: the version on the image may not reflect the [latest version of OneFilter/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+2. Now, use the checkboxes to select the categories to filter. Note that, once you have selected the categories, on the right table will show the existing families, with the category, family name, type name, Level and count columns.
```
Comment (ViniciusKawazoe) [inline docs/pages/OneFilter/OF-Levels.md:31]: - Capitalize "Categories"
- "Note that, once you have selected the Categories, ~~on~~ the right table will show the existing Families, with its Category, Family name, Type name, Level and count columns."

File: docs/pages/OneFilter/OF-Levels.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+![OneFilter choose categories](../../../assets\images\OneFilter\OF-Lv-Categories.gif)  
+<sub>Note: the version on the image may not reflect the [latest version of OneFilter/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+## Find Elements
+
+Create conditions rules to find elements between Levels and Reference Planes. Follow the steps:
+
+1. Select the AND/OR condition from the drop-down list
```
Comment (ViniciusKawazoe) [inline docs/pages/OneFilter/OF-Levels.md:40]: Quote indicated elements in the UI

"1. Select the "AND/OR" condition from the drop-down list"

File: docs/pages/OneFilter/OF-Levels.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+2. Click Add Rule
```
Comment (ViniciusKawazoe) [inline docs/pages/OneFilter/OF-Levels.md:42]: "2. Click "Add Rule""

File: docs/pages/OneFilter/OF-Levels.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+3. Select from the Reference level or Reference plane drop-down list
```
Comment (ViniciusKawazoe) [inline docs/pages/OneFilter/OF-Levels.md:44]: Capitalize Revit elements
- Reference Level
- Reference Plane

File: docs/pages/OneFilter/OF-Levels.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+The search box will search for elements contained in the filtered elements table.
+
+![OneFilter search](../../../assets\images\OneFilter\OF-Lv-Search.gif)  
+<sub>Note: the version on the image may not reflect the [latest version of OneFilter/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+---
+
+### Update or duplicate view
+
+Mark with checkbox elements that you want isolate with the Section Box. You have the ability to use existing or to duplicate 3D view. Visualize only filtered elements or all elements within Section Box with ability to define offset form marked elements.
```
Comment (ViniciusKawazoe) [inline docs/pages/OneFilter/OF-Levels.md:74]: "Check the elements that you want isolate within a Section Box. You can use existing 3D Views or duplicate them. Visualize filtered elements or all elements within the Section Box, with the ability to define an offset from marked elements."

File: assets/images/ViewManager/VM-Vr-BatchActions.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vr-BatchActions.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vr-Clear.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vr-Clear.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vr-Details.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vr-Details.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vr-ManualActions.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vr-ManualActions.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-DeleteProfile.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-DeleteProfile.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Profile.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Profile.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-ProfileImport.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-ProfileImport.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vt-BatchDelete.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vt-BatchDelete.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vt-BatchDuplicate.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vt-BatchDuplicate.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vt-Build.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vt-Build.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vt-Clear.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vt-Clear.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vt-EditViewTemplate.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vt-EditViewTemplate.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vt-Filter.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vt-Filter.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vt-QuickView.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vt-QuickView.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vt-Rename.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vt-Rename.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vt-deleteManual.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vt-deleteManual.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: assets/images/ViewManager/VM-Vt-duplicateManual.gif
Comment (ViniciusKawazoe) [inline assets/images/ViewManager/VM-Vt-duplicateManual.gif:1]: The GIFs were created using the outdated user interface. Please recreate the GIFs using the current UI.

File: docs/pages/ViewManager/VM-View-Templates.md [hunk 8]
```diff
@@ -90,15 +90,15 @@ Then define the Default Field Separator, Custom Field, Custom Separator, and add
...
 Use the drop-down lists to filter view templates by Disciplines and View Types.
 
-![View Templates filter view templates](../../../)
+![View Templates filter view templates](../../../assets\images\ViewManager\VM-Vt-Filter.gif)
 
 <sub>Note: the version on the image may not reflect the [latest version of ViewManager/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
 
 ### Quick visualization
 
 Visualize only the selected view templates, by clicking on Hide un-checked. You can also visualize only the view templates with associated views, by clicking on Has Associated Views.
```
Comment (ViniciusKawazoe) [inline docs/pages/ViewManager/VM-View-Templates.md:99]: "Visualize only the selected View Templates, by clicking on "Hide Unchecked". You can also visualize only the View Templates with associated Views, by clicking on "Has Associated Views"."

File: docs/pages/ViewManager/VM-View-Range.md [hunk 9]
```diff
@@ -16,24 +16,50 @@ Batch create Drawing Sheets, place Views based on a pre-defined template, and ea
 
 ---
 
-# View Templates tab
+# View Range tab
 
-Manage all View Templates in this tab.
+Manage the View Range of your views individually or in batch in one place.
```
Comment (ViniciusKawazoe) [inline docs/pages/ViewManager/VM-View-Range.md:21]: "Manage the View Range of your Views individually or in batches in one place."

File: docs/pages/ViewManager/VM-View-Range.md [hunk 10]
```diff
@@ -16,24 +16,50 @@ Batch create Drawing Sheets, place Views based on a pre-defined template, and ea
...
 
-## Rename
+## Manual actions
 
-View Manager also allows view templates to be renamed in batch. To do this, select the view templates you want and click to Rename. 
-Then define a Prefix, a Suffix, enter the name that will be replaced and finally the name that will replace it.
+1. Right-click on the view and click to open view.
```
Comment (ViniciusKawazoe) [inline docs/pages/ViewManager/VM-View-Range.md:25]: Always capitalize Revit elements, such as "Views", "View Templates", etc.

**Review by ViniciusKawazoe — CHANGES_REQUESTED** [review]

File: docs/pages/ViewManager/VM-Template-Transfer.md (hunk 1 above)
Comment (ElaineSimoes) [reply docs/pages/ViewManager/VM-Template-Transfer.md:29]: Fixed!

File: docs/pages/ViewManager/VM-View-Templates.md (hunk 8 above)
Comment (ElaineSimoes) [reply docs/pages/ViewManager/VM-View-Templates.md:99]: Fixed

File: docs/pages/ViewManager/VM-View-Range.md (hunk 9 above)
Comment (ElaineSimoes) [reply docs/pages/ViewManager/VM-View-Range.md:21]: Fixed

File: assets/images/ViewManager/VM-Tt-Destination.gif
Comment (ElaineSimoes) [reply assets/images/ViewManager/VM-Tt-Destination.gif:1]: Fixed!

File: assets/images/ViewManager/VM-Tt-EditViewTemplate.gif
Comment (ElaineSimoes) [reply assets/images/ViewManager/VM-Tt-EditViewTemplate.gif:1]: Fixed!

File: assets/images/ViewManager/VM-Tt-OriginTemplate.gif
Comment (ElaineSimoes) [reply assets/images/ViewManager/VM-Tt-OriginTemplate.gif:1]: Fixed!

File: assets/images/ViewManager/VM-Tt-Properties.gif
Comment (ElaineSimoes) [reply assets/images/ViewManager/VM-Tt-Properties.gif:1]: Fixed!

File: assets/images/ViewManager/VM-Vt-deleteManual.gif
Comment (ElaineSimoes) [reply assets/images/ViewManager/VM-Vt-deleteManual.gif:1]: Fixed!

File: assets/images/ViewManager/VM-Vt-duplicateManual.gif
Comment (ElaineSimoes) [reply assets/images/ViewManager/VM-Vt-duplicateManual.gif:1]: Fixed!

**Review by ViniciusKawazoe — APPROVED** [review]

**Pull request merged**